src/transformState.js

```diff
@@ -6,6 +6,29 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
I like that you decided to extract data from action object 👍 
But we can extract here all the data we will need in one line.

Reply:
I meant that instead of 3 lines of code, you could do it in one: 
`
const { type } = action;
const { extraData } = action;
const { keysToRemove } = action;
`

`
const { type, extraData, keysToRemove } = action;
`